recipes/ipydrawio/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set name = "ipydrawio" %}
+{% set version = "1.0.1" %}
+
+{% set build_number = 0 %}
+
+package:
+  name: {{ name }}-build
```

Comment:
Can you do a `s/{{ name }}/ipydrawio/g` for this file? It's hurting readability.

Reply:
a9534c2